### PR TITLE
added support for custompath for bufficons

### PR DESCRIPTION
**Pull request by Krypt0n**
for https://github.com/wurstscript/WurstStdlib2/issues/187

### CODE REVIEW

**Comment by Frotty** [comment]
That doesn't rly change the buff issue?

**Comment by Krypt0n** [comment]
why not?

**Comment by Frotty** [comment]
`createDummyBuffObject` uses `setIcon` and you cannot access the buff object afterwards?

**Comment by Krypt0n** [comment]
ah yeah, its cause i have custom function for createDummyBuffObject ,  not sure if you want to have another function for createDummyBuffObject  with overloaded paramters

**Comment by Frotty** [comment]
Which is why I suggested the solution I did.

**Comment by Krypt0n** [comment]
Yeah I know, just thought its not that clean. I mean its just a note, the whoel thing is not a big problem at all.

**Comment by Frotty** [comment]
Sure.
`function setIcon(string name)` should just reference the new one then tough.

**Comment by Krypt0n** [comment]
Not sure if you want to have it in a configurable member, or just as it is now, I do not really care^^

**Comment by Frotty** [comment]
I think it's fine, but should be added to hotdoc and perhaps other places? 🤧

**Comment by Frotty** [comment]
will you add this feature to the function's hotdoc?

**Comment by Krypt0n** [comment]
yeah sure, sry was away a few days

File: wurst/objediting/BuffObjEditing.wurst [hunk 1]
```diff
@@ -74,9 +74,13 @@ public class BuffDefinition
 	function setIconNormal(int level, string value)
 		def.setLvlDataString("fart",  level, 0,  value)
 
+	/** 
+	 * Set the icon's path for the buff obj
+     * @param string name - name of the icon file, add "@!!" as prefix to overide default path by custom path
```
Comment (Frotty) [inline wurst/objediting/BuffObjEditing.wurst]: `@param string name`
This is no part of the hotdoc specification. Please remove and use normal description.

**Comment by Frotty** [comment]
Alright, this can't hurt imho. 👍

**Pull request merged**